Cannot find module "protobuf/minimal", "Did you mean to import protobufjs/minimal.js?"

protobuf.js version: 7.2.0

Trying to use protobufjs in an ES module.

```
{
	"name": "myproject",
	"version": "1.0.0",
	"type": "module",
	"dependencies": {
		"protobufjs": "^7.2.0"
	},
	"devDependencies": {
		"protobufjs-cli": "^1.1.0",
		"typescript": "^4.9.5"
	}
}
```

I generated the protobuf messages module (and typings) like this

```
npx pbjs -t static-module -w es6 protobuf/myproject.proto -o src/protobuf/protobuf.js
npx pbts src/protobuf/protobuf.js -o src/protobuf/protobuf.d.ts
```

At the top of the `src/protobuf/protobuf.js` file I see this:

```
/*eslint-disable block-scoped-var, id-length, no-control-regex, no-magic-numbers, no-prototype-builtins, no-redeclare, no-shadow, no-var, sort-vars*/
import * as $protobuf from "protobufjs/minimal";
```

Given a simple program

```
// src/main.ts
import { SomeMessage } from "src/protobuf.js"
console.log("Hello world!")
```

And compiling with this `tsconfig.json`:

```
{
	"compilerOptions": {
		"module": "NodeNext",
		"moduleResolution": "nodenext",
		"target": "ES2022",
		"sourceMap": true,
		"strict": true,
		"skipLibCheck": true,
		"outDir": "build",
		"noImplicitAny": false,
		"allowJs": true
	},
	"include": [
		"src/**/*",
		"tests/**/*"
	]
}
```

When I try to run the project `node --enable-source-maps build/main.js` I get this error message:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/birger/myproject/node_modules/protobufjs/minimal' imported from /home/birger/myproject/build/src/protobuf/protobuf.js
Did you mean to import protobufjs/minimal.js?
```

Tbh I don't know if I correctly understood the process of generating the protobuf module with typings. Should I not be using the `-w es6` flag?

--- a/cli/targets/static-module.js
+++ b/cli/targets/static-module.js
@@ -18,7 +18,11 @@ function static_module_target(root, options, callback) {
             return;
         }
         try {
-            output = util.wrap(output, protobuf.util.merge({ dependency: "protobufjs/minimal" }, options));
+            output = util.wrap(output, protobuf.util.merge({
+                dependency: options.wrap === "es6"
+                    ? "protobufjs/minimal.js"
+                    : "protobufjs/minimal"
+            }, options));
         } catch (e) {
             callback(e);
             return;

--- a/cli/wrappers/es6.js
+++ b/cli/wrappers/es6.js
@@ -1,4 +1,4 @@
-import * as $protobuf from $DEPENDENCY;
+import $protobuf from $DEPENDENCY;
 
 $OUTPUT;
 

--- a/tests/cli.js
+++ b/tests/cli.js
@@ -93,6 +93,23 @@ tape.test("pbjs generates static code", function(test) {
     });
 });
 
+tape.test("pbjs generates correct ES6 static-module imports", function(test) {
+    cliTest(test, function() {
+        var root = protobuf.loadSync("tests/data/cli/test.proto");
+        root.resolveAll();
+
+        var staticModuleTarget = require("../cli/targets/static-module");
+
+        staticModuleTarget(root, {
+            wrap: "es6",
+        }, function(err, jsCode) {
+            test.error(err, "static-module code generation worked");
+            test.ok(jsCode.includes("import $protobuf from \"protobufjs/minimal.js\";"), "es6 wrapper uses a default import and explicit .js extension");
+            test.end();
+        });
+    });
+});
+
 tape.test("without null-defaults, absent optional fields have zero values", function(test) {
     cliTest(test, function() {
         var root = protobuf.loadSync("tests/data/cli/null-defaults.proto");

